Dockerfile

```diff
@@ -20,6 +20,11 @@ RUN cp /openmrs_distro/distro/target/sdk-distro/web/openmrs-distro.properties /o
 RUN cp -R /openmrs_distro/distro/target/sdk-distro/web/modules /openmrs/distribution/openmrs_modules
 RUN cp -R /openmrs_distro/distro/target/sdk-distro/web/owa /openmrs/distribution/openmrs_owas
 
+#Include Custom Modules for Namibia Distro
+RUN ls la
+RUN cp  /openmrs_distro/initializer-2.6.0-SNAPSHOT.omod /openmrs/distribution/openmrs_modules
```

Comment:
this needs to chance from RUN cp /openmrs_distro/initializer-2.6.0-SNAPSHOT.omod /openmrs/distribution/openmrs_modules to RUN cp /openmrs_distro/distro/initializer-2.6.0-SNAPSHOT.omod /openmrs/distribution/openmrs_modules